webgl-examples/tutorial/sample5/index.html

```diff
@@ -3,14 +3,14 @@
   <head>
     <meta charset="utf-8" />
     <title>WebGL Demo</title>
-    <link rel="stylesheet" href="../webgl.css" type="text/css" />
+    <link rel="stylesheet" href="./webgl.css" type="text/css" />
```

Comment:
I've removed these. I could not see a need for this CSS although I was nervous getting rid of it in case I was missing something. I'd rather not include it though as it's just extra instructions for people to follow.